app/operations/increment_score.rb

```diff
@@ -36,7 +35,8 @@ def user
     @user ||= User.find(user_id)
   end
 
-  def to_user_timezone(t)
-    t.in_time_zone(user.time_zone)
+  def now_in_user_zone
+    zone = user.time_zone ? ActiveSupport::TimeZone[user.time_zone] : Time.zone
```

Comment:
I think you could simplify as `ActiveSupport::TimeZone[user.time_zone] || Time.zone`, this way we would also handle invalid time zones.